doc/about.rst

```diff
@@ -96,86 +96,176 @@ following paper:
+<https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos>`_
+source directory.
 
 .. image:: images/scikit-learn-logo-notext.png
    :align: center
 
 Funding
 -------
+Scikit-Learn is a community driven project, however institutional and private
+grants help to assure its sustainability.
 
-`INRIA <https://www.inria.fr>`_ actively supports this project. It has
-provided funding for Fabian Pedregosa (2010-2012), Jaques Grobler
-(2012-2013) and Olivier Grisel (2013-2017) to work on this project
-full-time. It also hosts coding sprints and other events.
+The project would like to thank the following funders. 
 
-.. image:: images/inria-logo.jpg
-   :width: 200pt
-   :align: center
-   :target: https://www.inria.fr
+...................................
 
-`Paris-Saclay Center for Data Science <https://www.datascience-paris-saclay.fr/>`_
-funded one year for a developer to work on the project full-time
-(2014-2015) and 50% of the time of Guillaume Lemaitre (2016-2017).
+The `Scikit-Learn Consortium at Inria Foundation
+<https://scikit-learn.fondation-inria.fr/en/home/>`_ `Members 
+<https://scikit-learn.fondation-inria.fr/en/home/#sponsors>`_ fund Olivier
+Grisel, Guillaume Lemaitre, Jérémie Du Boisberranger and Chiara Marmo.
 
-.. image:: images/cds-logo.png
-   :width: 200pt
-   :align: center
-   :target: https://www.datascience-paris-saclay.fr/
+**Platinum Members**
 
-`NYU Moore-Sloan Data Science Environment <https://cds.nyu.edu/mooresloan/>`_
-funded Andreas Mueller (2014-2016) to work on this project. The Moore-Sloan Data Science
-Environment also funds several students to work on the project part-time.
 
-.. image:: images/nyu_short_color.png
-   :width: 200pt
+.. figure:: images/microsoft.png
+   :width: 100pt
    :align: center
-   :target: https://cds.nyu.edu/mooresloan/
+   :target: https://www.microsoft.com/
 
+.. figure:: images/bcg.png
+   :width: 100pt
+   :align: center
+   :target: https://www.bcg.com/beyond-consulting/bcg-gamma/default.aspx
 
-`Télécom Paristech <https://www.telecom-paristech.fr/>`_ funded Manoj Kumar (2014),
-Tom Dupré la Tour (2015), Raghav RV (2015-2017), Thierry Guillemot (2016-2017)
-and Albert Thomas (2017) to work on scikit-learn.
+**Gold Members**
 
-.. image:: themes/scikit-learn/static/img/telecom.png
+.. figure:: images/axa.png
+   :width: 50pt
+   :align: center
+   :target: https://www.axa.fr/
+
+.. figure:: images/bnp.png
    :width: 100pt
    :align: center
-   :target: https://www.telecom-paristech.fr/
+   :target: https://www.bnpparibascardif.com/
 
+.. figure:: images/intel.png
+   :width: 70pt
+   :align: center
+   :target: https://www.intel.com/
 
-`Columbia University <https://columbia.edu/>`_ funds Andreas Müller since 2016.
+.. figure:: images/nvidia.png
+   :width: 70pt
+   :align: center
+   :target: https://www.nvidia.com/
 
-.. image:: themes/scikit-learn/static/img/columbia.png
-   :width: 100pt
+**Silver Members**
```

Comment:
We should find a way to scope those sections under the "scikit-learn consortium at Inria Foundation" group more explicitly.

Otherwise one gets the impression that the following supporters (Columbia U, Sloan, U. of Sydney...) are silver level members of the  Scikit-learn consortium at Inria Foundation.

Reply:
This is probably equivalent to what @jnothman expressed in his comment:

> one area to improve: I think it's easy to currently misconstrue that the consortium is the primary/only funding source. So a preface might help clarify that several sources will be listed.